app/scripts/proactive/view/WorkflowView.js

```diff
@@ -509,7 +509,7 @@ define(
                 return (name === view.model.get("Task Name"));
             })
         },
-        copyPasteTasks: function (tasks, position) {
+        copyPasteTasks: function (tasks, position , newTaskModels, Taks) {
```

Comment:
I don't understand what is `Taks`, maybe you could rename it into something more explicit. I think that you should follow the naming convention and put the variable in lowercase.

Reply:
looks like a typo, but yes it should be more understandable. for example tasksToCopy, tasksFromCurrentModel (I don't know the precise semantic here)

Reply:
done